tests/integration/test_logcollector/test_options/test_restrict_regex.py

```diff
@@ -0,0 +1,269 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The 'wazuh-logcollector' daemon monitors configured files and commands for new log messages.
+       Specifically, these tests check the behavior of the restrict and ignore options, that allow
+       users to configure regex patterns that limit if a log will be sent to analysis or will be ignored.
+       The restrict causes any log that does not match the regex to be ignored, conversely, the 'ignore' option
+       causes logs that match the regex to be ignored and not be sent for analysis.
+
+components:
+    - logcollector
+
+suite: options
```

Comment:
The suite name could be the: logs filter options.

Reply:
fixed in https://github.com/wazuh/wazuh-qa/pull/3582/commits/736cebbf8d76781caa2709ed9b8f305e31183c79